test/openAPI/features/client_create.feature

```diff
@@ -1,16 +1,132 @@
+    When User sends POST request with given requestTime, "<clientId>" as clientId, "<clientName>" as clientName, "<relyingPartyId>" as relyingPartyId, "<logoUri>" as logoUri, "<publicKey>" as publicKey, "<authContextRefs>" as authContextRefs, "<userClaims>" as userClaims, "authorization_code" as grantTypes, "private_key_jwt" as clientAuthMethods, "<redirectUris>" as redirectUris
+    Then User receives a response from the POST /client-mgmt/oidc-client endpoint
+    And The POST /client-mgmt/oidc-client endpoint response should be returned in a timely manner 15000ms
+    And The POST /client-mgmt/oidc-client endpoint response should have status 200
+    And The POST /client-mgmt/oidc-client endpoint response should have content-type: application/json header
+    And The POST /client-mgmt/oidc-client endpoint response should match json schema
+    And The POST /client-mgmt/oidc-client endpoint response should contain "<clientId>" as clientId
+    And The POST /client-mgmt/oidc-client endpoint response should contain empty errors array
 
-  Request endpoint: POST /client-mgmt/oidc-client
+    Examples: Valid data
+    | clientId           | clientName       | relyingPartyId | logoUri             | publicKey                | authContextRefs                    | userClaims         | redirectUris           |
+    | payment-service    | Payment Service  | abc-gov        | http://example1.com | bXlfcHVibGljX2tleV8wMQ== | idbb:acr:static-code               | given_name         | http://redirectMe1.com |
+    | e-bank-service     | Bank Service     | bankser-gov    | http://example2.com | aGlzX3B1YmxpY19rZXlfMDI= | idbb:acr:linked-wallet-static-code | family_name        | http://redirectMe2.com |
+    | e-commerce-service | Commerce Service | ecommer-gov    | http://example3.com | aGVyX3B1YmxpY19rZXlfMDE= | idbb:acr:biometrics-generated-code | middle_name        | http://redirectMe3.com |
+    | health-service     | Health Service   | health-gov     | http://example4.com | cHVibGljX2tleV8wMQ==     | idbb:acr:biometrics                | preferred_username | http://redirectMe4.com |
 
-  Background: 
+  @unit @negative
+  Scenario: Not able to add the new client to the Open ID Connect (OIDC) because of invalid requestTime
     Given The user wants to add the new client to the Open ID Connect (OIDC)
+    When User sends POST request with given "2011-10-05" as invalid requestTime
+    Then User receives a response from the POST /client-mgmt/oidc-client endpoint
+    And The POST /client-mgmt/oidc-client endpoint response should be returned in a timely manner 15000ms
+    And The POST /client-mgmt/oidc-client endpoint response should have status 200
+    And The POST /client-mgmt/oidc-client endpoint response should have content-type: application/json header
+    And The POST /client-mgmt/oidc-client endpoint response should match json schema with error
+    And The POST /client-mgmt/oidc-client endpoint response should match with error code "invalid_input"
```

Comment:
For this scenario the expected error code is invalid_request, as the new PR https://github.com/GovStackWorkingGroup/bb-identity/pull/36